akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -327,83 +327,159 @@ class ReplicatedDataSerializer(val system: ExtendedActorSystem)
     }
   }
 
-  def ormapToProto(ormap: ORMap[_]): rd.ORMap = {
+  def ormapToProto(ormap: ORMap[_, _]): rd.ORMap = {
     val b = rd.ORMap.newBuilder().setKeys(orsetToProto(ormap.keys))
-    ormap.entries.toVector.sortBy { case (key, _) ⇒ key }.foreach {
-      case (key, value) ⇒ b.addEntries(rd.ORMap.Entry.newBuilder().
-        setKey(key).setValue(otherMessageToProto(value)))
+    ormap.entries.toVector.foreach {
```

Comment:
I didn't realize that. I changed the implementation to sort in deterministic order. I also made the implementation more generic, because all maps are more or less the same. Do you like this approach with Type classes?